dd-sdk-android/src/main/kotlin/com/datadog/android/v2/core/internal/storage/BatchConfirmation.kt

```diff
@@ -6,6 +6,9 @@
 
 package com.datadog.android.v2.core.internal.storage
 
-internal fun interface BatchConfirmation {
```

Comment:
are we removing functional interface property here (and in the other interfaces), because it is complicated to get annotation in case of lamba implementation of the functional interface?